app/emotion_service.py

```
@@ -0,0 +1,117 @@
+        except Exception as e:
+            return {
+                "error": f"분석 중 오류 발생: {str(e)}",
+                "emotion": "unknown",
+                "confidence": 0.0
+            }
+        finally:
+            # 임시 파일 정리
+            try:
+                import os
+                os.unlink(tmp_file_path)
+            except:
```

Ruff (0.14.1)

Do not use bare `except`

(E722)